plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/InternalDeltaLakeConnectorFactory.java

```diff
@@ -71,7 +72,8 @@ public static Connector createConnector(
             String catalogName,
             Map<String, String> config,
             ConnectorContext context,
-            Module module)
+            Module module,
+            Optional<HiveMetastore> metastore)
```

Comment:
add before `Module module,` (as you did in `TestingDeltaLakePlugin` class)

Reply:
actually, i think the change here is not required.
You can bind your own metastore via `Module module`, you just need to set `hive.metastore` to some unsupported value (eg `other`).

would that work? i think this is what `TestDeltaLakeMetastoreAccessOperations` is doing.